packages/pglite/src/templating.ts

```diff
@@ -0,0 +1,188 @@
+enum TemplateTagType {
+  identifier,
+  raw,
+}
```

Comment:
huh - interesting read, I'll get rid of this template tag type altogether since the current code doesn't use it (it spawned out of an initial thought on how to do this)